app/src/main/java/com/github/se/eventradar/ui/event/EventDetails.kt

```diff
@@ -30,210 +30,137 @@ import androidx.compose.ui.text.font.FontWeight
 private val contentTextSize = 14.sp
 
-// Temporary colors scheme of the screen
-private val DT_fieldTitleColor = Color(0xFF79747E)
-private val DT_fieldContentColor = Color(0xFFFFFFFF)
-private val DT_titleColor = Color(0xFFFFFFFF)
-
-private val DT_backgroundColor = Color(0xFF1B191B) // TODO take the actual from figma
-private val DT_lightBackgroundColor = Color(0xFF24292F)
-
-private val DT_special1Color = Color(0xFFAA23AA)
-
-
-private val eventImage = R.drawable.ic_launcher_background
-
 @Composable
-// TODO new header for nav and viewModel
-// fun EventDetails(EventviewViewModel: ViewModel = viewModel(), navigationActions: NavigationActions){
-fun EventDetails() {
-
-
-    Column(
-        modifier = Modifier.fillMaxHeight(),
-        verticalArrangement = Arrangement.SpaceBetween
-    ) {
-        Image(
-            painter = painterResource(eventImage),
-            contentDescription = "Event Image",
-            modifier = Modifier
-                .fillMaxWidth()
-                .height(imageHeight),
-            contentScale = ContentScale.FillWidth
-
-        )
-
-        // go back button
-        Button(
-            onClick = { /*TODO*/ },
-            modifier = Modifier
-                .wrapContentSize()
-                .align(Alignment.Start)
-                .testTag("backButton"),
-            colors =
+fun EventDetails(navigationActions: NavigationActions) {
+
+  val fieldTitleColor = MaterialTheme.colorScheme.onSurfaceVariant
+  val fieldContentColor = MaterialTheme.colorScheme.onSurface
+
+  // Temporary text field values to be replaced with some ViewModel.stateVar.collectAsState()
+  val eventImage = R.drawable.ic_launcher_background
+  val eventTitle = "Event Title"
+  val headerDescription = "Description"
+  val headerDistance = "Distance from you"
+  val headerDate = "Date"
+  val headerCategory = "Category"
+  val headerTime = "Time"
+  val contentDescription =
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc et ornare  dui. Integer convallis purus odio, vitae mattis erat ultricies non.  Donec et magna hendrerit, molestie lorem vel, facilisis augue. "
+  val contentDistance = "xx km"
+  val contentDate = "dd/mm/yyyy"
+  val contentCategory = "Cat x"
+  val contentTime = "xx:xx"
+
+  Column(modifier = Modifier.fillMaxHeight(), verticalArrangement = Arrangement.SpaceBetween) {
+    Image(
+        painter = painterResource(eventImage),
+        contentDescription = "Event Image",
+        modifier = Modifier.fillMaxWidth().height(imageHeight),
+        contentScale = ContentScale.FillWidth)
+
+    // Go back button
+    Button(
+        onClick = { navigationActions.goBack() },
+        modifier = Modifier.wrapContentSize().align(Alignment.Start).testTag("backButton"),
+        colors =
             ButtonDefaults.buttonColors(
-                contentColor = DT_titleColor,
+                contentColor = MaterialTheme.colorScheme.surface,
                 containerColor = Color(0x00FFFFFF),
```

Comment:
nit: Make sure to change this to primaryContainer when the Color and Material PR has been merged :)